TrackingTools/PatternTools/interface/Trajectory.h

```diff
@@ -240,6 +302,12 @@ class Trajectory
 
   int lostHits() const { return theNumberOfLostHits;}
 
+  /** Number of hits that are not compatible with the CCC used during
+   *  patter recognition. Used mainly as a criteria for abandoning a
+   *  trajectory candidate during trajectory building.
+   */
+  int CCCBadHits() const { return theNumberOfCCCBadHits_;}
```

Comment:
as in TempTrajectory:
start with a lower case
``` cccBadHits``` is just as good
... in general, I find it a bad idea to define a category based on one particular cut. If we redo this cut to do something else (not just cluster charge), the name will become inappropriate.